### ISSUE TITLE
"An error occurred on the server while streaming results." error returned irrespective of input.

### ISSUE THREAD

**Issue by TomKimber** [microsoft/GitHub-Copilot-for-Azure#16]
I receive the error "An error occurred on the server while streaming results." irrespective of the input. I've tried switching subscriptions, uninstalling/reinstalling extension, re-authenticating, etc but still receive the same error. All other Copilot/Copilot Chat functionality is working as expected. 

GitHub Copilot Chat Output:
2024-11-07 10:37:10.146 [info] message 0 returned. finish reason: [stop]
2024-11-07 10:37:10.146 [info] request done: requestId: [2f0aef41-ceb5-4a57-b44a-fd009c1d544e] model deployment ID: []

**Comment by bwateratmsft** [comment]
Can you share the `@azure` Output log?

**Comment by TomKimber** [comment]
@bwateratmsft 
@azure Output:
2024-11-07 14:07:54.387 [info] Created chat session ID: 19785e2c-617d-49d7-8f78-3539b9dcd008
2024-11-07 14:07:54.913 [error] [Microsoft.Azure.CopilotForAzure.AIService.Orchestration.OrchestratorBase] Error while processing request.

**Comment by bwateratmsft** [comment]
Well that's not much ☹ 

If you set the log level to trace (by clicking the gear icon next to the output log dropdown), does it show any more?
![Image](https://github.com/user-attachments/assets/b748eced-e02e-495f-88b5-dfad9e078a15)

**Comment by bwateratmsft** [comment]
Do you have multiple GitHub accounts logged in?

Can you try signing out of your GitHub account(s) and try chatting again? I'm wondering if it's trying to use the wrong account to communicate with the Copilot backend.

![Image](https://github.com/user-attachments/assets/d764ab10-0187-43cb-8ee2-438dd37ae09f)

**Comment by TomKimber** [comment]
I did have multiple GitHub accounts logged in. I logged out and logged back in only with the account associated with the Copilot subscription, tested, `@azure` worked. I then logged back in to the other two GitHub accounts and `@azure` is still working. I did try logging out og the account with the Copilot subscription prior to raise the issue but may not have logged out of all the GitHub accounts. Thanks for your assistance with resolving the issue!
